README.md

```diff
@@ -1,21 +1,52 @@
+| 2   | [classifier_comparison.ipynb](classifier_comparison.ipynb)                                     | Demonstrate how cleanlab can be used to train 10 different classifiers on 4 dataset distributions with label errors.                                                                                                                                                         |
+| 3   | [model_selection_demo.ipynb](model_selection_demo.ipynb)                                       | Perform hyperparameter optimization to find the best settings of cleanlab's optional parameters.                                                                                                                                                                             |
+| 4   | [simplifying_confident_learning_tutorial.ipynb](simplifying_confident_learning_tutorial.ipynb) | Implement cleanlab as raw numpy code.                                                                                                                                                                                                                                        |
+| 5   | [visualizing_confident_learning.ipynb](visualizing_confident_learning.ipynb)                   | Demonstrate how cleanlab performs noise matrix estimation.                                                                                                                                                                                                                   |
+| 6   | [cifar10-cnn-coteaching](cifar10-cnn-coteaching)                                               | Demonstrate the use of two experimental modules from cleanlab: [cifar_cnn.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/cifar_cnn.py) and [coteaching.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/coteaching.py) |
+| 7   | [mnist-cnn](mnist-cnn)                                                                         | Demonstrate the use of the following experimental module from cleanlab: [mnist_pytorch.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/mnist_pytorch.py)                                                                                          |
+| 8   | [amazon-reviews-fasttext](amazon-reviews-fasttext)                                             | Demonstrate the use of the following experimental module from cleanlab: [fasttext.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/fasttext.py)                                                                                                    |
+
+## Instructions
+
```

Comment:
should specify which version of cleanlab is recommended here (see slack discussion).

Reply:
Updated instructions for user to use the latest stable cleanlab release. Also updated the `requirements.txt` file so it just installs `cleanlab` instead of pointing to master branch.